src/transactions/AllowTrustOpFrame.h

```diff
@@ -23,21 +23,26 @@ class AllowTrustOpFrame : public TrustlineOpFrameBase
 
     void setResultSelfNotAllowed() override;
     void setResultNoTrustLine() override;
+    void setResultLowReserve() override;
     void setResultSuccess() override;
     bool isAuthRevocationValid(AbstractLedgerTxn& ltx,
                                bool& authRevocable) override;
-    bool isAuthRevocationValid(bool authRevocable, const LedgerTxnEntry& trust,
-                               uint32_t flags) override;
+    bool isRevocationToMaintainLiabilitiesValid(bool authRevocable,
+                                                LedgerTxnEntry const& trust,
+                                                uint32_t flags) override;
+
+    LedgerKey getTrustlineKey() const override;
+    AccountID const& getOpTrustor() const override;
+    Asset const& getOpAsset() const override;
```

Comment:
I can't comment on line 20, but `mAsset` is no longer needed with this.

Reply:
Why is it no longer needed? The `mAllowTrust.asset` is an `AssetCode` and we construct `mAsset` from that.

Reply:
You are correct, I was thinking of `SetTrustLineFlagsOp` which contains an `Asset`.